delivery_carrier_return_barcode_pattern/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Delivery Carrier Return Barcode Pattern",
+    "summary": "Allow to define a return barcode pattern on delivery methods",
```

Comment:
This feature could be used outside of a return workflow I guess?
This is all about matching a barcode originated from a carrier, should we enforce the use of "Return" here?

Reply:
It is for processing what comes back from the carrier, so somehow yes